ci/prs.yml

```diff
@@ -73,6 +73,14 @@ jobs:
             exit 1
         fi
 
+- job: check_compatibility_protobuf
```

Comment:
Note that the job is already running on PRs, because `build.yml` is common to both PR builds and main builds. Therefore, as it stands, this would run it twice on PRs.